README.md

```diff
@@ -139,6 +139,12 @@ You can also specify separate download roots for npm and node as they are now st
 </plugin>
 ```
 
+By default node/npm will be cached to to the Maven local repository. To disable this and use the `workingDirectory/cache` add
```

Comment:
I believe by default it now downloads the npm (etc) tar.gz to e.g. .m2 directory instead of a temporary directory. The working directory is where you are running the build from. I guess we could make this more configurable? But I just wanted to the option to make it .m2 or something else :-)